bob.spec

```diff
@@ -20,58 +18,61 @@ Requires: make-gnu >= 4.2-2
-- Flexibility. Most objects defined to Bob typically build using your default values. Have a program that requires a custom activation group or a data area that needs to be created with a certain value? No problem, overriding compile parameters is trivial, and writing custom recipes for special objects is very straightforward. If you can code it, you can build it.
-- Ease of use. Invoking a build of an entire codebase is done with just a single command. Or, if the Rational Developer for i integration pieces are installed, a single button click.
+- Speed. Bob only compiles objects that need recompiling, like from new or 
+  changed source code.
+- Reliability. Bob understands the relationships between your objects, so if an
+  item changes, then it and everything depending on it will be rebuilt.
+- Industry standard. Object dependencies are specified using standard makefile
+  syntax, and the actual build engine is GNU Make -- exactly like tens of 
+  thousands of Linux and Unix software projects.
+- Flexibility. Most objects defined to Bob typically build using your default
+  values. Have a program that requires a custom activation group or a data area
+  that needs to be created with a certain value? No problem, overriding compile
+  parameters is trivial, and writing custom recipes for special objects is very
+  straightforward. If you can code it, you can build it.
+- Ease of use. Invoking a build of an entire codebase is done with just a 
+  single command. Or, if the Rational Developer for i integration pieces are
+  installed, a single button click.
 
 
 %prep
 %setup -n ibmi-bob-%{version}
-tar -xzvf %{SOURCE1}
 
 
 %build
-cd CRTFRMSTMF-master && %{_bindir}/gmake && cd ..
-
-system 'DLTF FILE(CRTFRMSTMF/CRTFRMSTMF)' || true
-system 'CRTSAVF FILE(CRTFRMSTMF/CRTFRMSTMF)'
-system "SAVOBJ OBJ(CRTFRMSTMF) LIB(CRTFRMSTMF) DEV(*SAVF) OBJTYPE(*PGM *PNLGRP *CMD) SAVF(CRTFRMSTMF/CRTFRMSTMF)"
-rm -f ./crtfrmstmf.savf
-system "CPYTOSTMF FROMMBR('/QSYS.LIB/CRTFRMSTMF.LIB/CRTFRMSTMF.FILE') TOSTMF('./crtfrmstmf.savf')" || true
+echo "Skip build"
 
 %install
 mkdir -p %{buildroot}%{_libdir}/bob
 mkdir -p %{buildroot}%{_bindir}/
 cp -r ./* %{buildroot}%{_libdir}/bob
-ln -sf %{_libdir}/bob/makei %{buildroot}%{_bindir}/makei
-ln -sf %{_libdir}/bob/launch %{buildroot}%{_bindir}/launch
-
-
-%post
-echo "Installing CRTFRMSTMF..."
-system "CRTLIB LIB(CRTFRMSTMF)" || true
-system 'DLTF FILE(CRTFRMSTMF/CRTFRMSTMF)' || true
-system "CPYFRMSTMF FROMSTMF('%{_libdir}/bob/crtfrmstmf.savf') TOMBR('/QSYS.LIB/CRTFRMSTMF.LIB/CRTFRMSTMF.FILE')"
-system "RSTOBJ OBJ(CRTFRMSTMF) SAVLIB(CRTFRMSTMF) DEV(*SAVF) OBJTYPE(*PGM *PNLGRP *CMD) SAVF(CRTFRMSTMF/CRTFRMSTMF)"
+ln -sf %{_libdir}/bob/scripts/makei %{buildroot}%{_bindir}/makei
+ln -sf %{_libdir}/bob/scrpts/launch %{buildroot}%{_bindir}/launch
+ln -sf %{_libdir}/bob/scrpts/crtfrmstmf %{buildroot}%{_bindir}/crtfrmstmf
```

Comment:
I think you misspelled scripts as scrpts

Reply:
I think you are viewing a previous commit here